resources/views/components/package.blade.php

```diff
@@ -1,5 +1,5 @@
-<div class="w-full lg:w-1/2 px-4 pb-8">
-    <div class="bg-astro-moonlight p-8 rounded overflow-hidden">
+<div class="px-4 pb-8">
+    <div class="bg-astro-moonlight p-8 rounded overflow-hidden h-full">
```

Comment:
One last thing - adding a `flex flex-col` here and a `flex-grow` to the description `<p>` tag should force the link to stay at the bottom - even if the description isn't the same height as the other one.
https://play.tailwindcss.com/uqr2UuIEUW

Reply:
Yeah that's what I needed!